doc/geos.sgml

```diff
@@ -332,6 +332,14 @@ The <tt/parameter/ is the field width in pixels (range 1-31) and the mode bits.
 the string can be filled with zeroes (the string is always 5 characters long) or not and left or right
 justified to the given pixel. See <tt/ggraph.h/ for predefined values for <tt/parameter/.
 
+<sect3>SetNewMode
```

Comment:
Are you sure this belongs to the section "Character and string output" ?

Reply:
That was a tough call. I wasnt quite sure where to put it, honestly.  Im open to suggestions.

Reply:
I see. Please move it to the very beginning of the *Graphics* section (right before *SetPattern*).

Reply:
done in latest.  I just happen to be wide awake with the code sitting in front of me :)  Gonna tackle greg's suggestion on the DA stuff in a bit.